docs/requirement-sets/outlook/requirement-set-1.5/outlook-requirement-set-1.5.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Outlook add-in API requirement set 1.5
 description: Features and APIs that were introduced for Outlook add-ins and the Office JavaScript APIs as part of Mailbox API 1.5.
-ms.date: 05/17/2021
+ms.date: 04/05/2022
 ms.localizationpriority: medium
 ---
 
```

Comment:
In the "What's new..." section, consider noting something like "Added support for Office.context.diagnostics property and related objects."